airbyte-webapp/src/components/base/Input/Input.tsx

```diff
@@ -89,6 +89,7 @@ export const Input: React.FC<InputProps> = ({ light, error, ...props }) => {
           {
             [styles.disabled]: props.disabled,
             [styles.password]: isPassword,
+            "fs-exclude": isPassword,
```

Comment:
Could we maybe make this class an export from some (ts) file, to have it more verbose in those places what it does? Alternatively I'd suggest we add a comment here, so that if we read that in the future, won't need to google what that class does :)